content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -0,0 +1,178 @@
++++
+title = "Pipelines Standalone Deployment in GCP"
+description = "Instructions to deploy Kubeflow Pipelines standalone to a cluster"
+weight = 15
++++
+
+As an alternative to [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize
+manifests in GCP.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+## Deploying Kubeflow Pipelines standalone to an existing cluster
+
+1. Deploy latest version of Kubeflow Pipelines:
+
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
+kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+```
+
+2. Get the URL for the Kubeflow Pipelines UI :
+
+```
+kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com
+```
+
+## Deploying Kubeflow Pipelines standalone from scratch
+
+1. Prepare a Kubernetes cluster:
+
+   You can reference [Creating a cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster) for Google Cloud Platform (GCP).
+
+   Recommend using the following gcloud command to create a cluster that can run all pipeline samples:
+
+   ```
+   # The following parameters can be customized based on your needs.
+   CLUSTER_NAME="kubeflow-pipelines-standalone"
+   # You can find available zones in https://cloud.google.com/compute/docs/regions-zones/#available
+   ZONE="us-central1-a"
+   MACHINE_TYPE="n1-standard-2" # 2 CPU and 7.50GB memory
+   SCOPES="storage-rw,cloud-platform" # These scopes are needed for running some pipeline samples
+   gcloud container clusters create $CLUSTER_NAME \
+       --zone $ZONE \
+       --machine-type $MACHINE_TYPE \
+       --scopes $SCOPES \
+       --num-nodes 2 \
+       --max-nodes 5 \
+       --min-nodes 2 \
+       --enable-autoscaling
+   ```
+
+   Reference:
+
+   - Get gcloud CLI tool at https://cloud.google.com/sdk/gcloud/.
+   - Read `gcloud container clusters create` command documentation at https://cloud.google.com/sdk/gcloud/reference/container/clusters/create.
+
+1. Download kubectl CLI tool:
+
+   You can get kubectl from [its official doc](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+   Requirement: kubectl >=1.14 for native support of kustomize (the `kubectl apply -k` options you will see below).
```

Comment:
For clarity, it's best to write this out in full, rather than abbreviate it. So, change this:

```
Requirement: kubectl >=1.14 for ...
```

To this:

```
You need kubectl version 1.14 or later, for ...
```

Reply:
Thanks, your wording does sound better!